src/mainboard/opentitan/crb/link.ld

```diff
@@ -30,6 +30,10 @@ SECTIONS
     . = 0x20000000;
     .bootblock :
     {
+        /* Ensure that the flash header matches what the OpenTitain ROM expects.
```

Comment:
oh damn. OpenTitan not OpenTitain

Reply:
Argh! Fixed here: https://github.com/oreboot/oreboot/pull/336